python/dalex/dalex/_arena/plots/_feature_importance_container.py

```diff
@@ -0,0 +1,35 @@
+from .._plot_container import PlotContainer
+
+class FeatureImportanceContainer(PlotContainer):
+    info = {
+        'name': "Variable Importance",
+        'plotType': "FeatureImportance",
+        'plotCategory': "Dataset Level",
+        'requiredParams': ["model"]
+    }
+    options = {
+        'N': { 'default': None, 'desc': 'Number of observations to use. None for all.' },
+        'B': { 'default': 10, 'desc': 'Number of permutation rounds to perform each variable' }
```

Comment:
Do we want `Explainer` defaults? (`model_parts` uses `N=1000`, `B=10`)

I guess it is a question for more default parameters.

Reply:
I try to keep same options in R and Python versions of Arena. If user runs both with unchanged options, then plots from Python and R should be comparable.